### PR TITLE
Added show command and some fixes

### LINKED ISSUE #263: [PE-D] allowing for test name to have extra field
-------------
Labels: `type.FeatureFlaw` `severity.Medium`
original: JaCh23/ped#3

### LINKED ISSUE #264: [PE-D] No way to view input again after input is entered
Currently, after the user inputs via the `input` command, there is no way for the user to review the input again.

-------------
Labels: `type.FeatureFlaw` `severity.Low`
original: alwinangys/ped#4

### PR DESCRIPTION
**Pull request by Kair0s3**
Fixes #263 
Fixes #264

### CODE REVIEW

File: src/main/java/decodex/commands/ShowCommand.java [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+    @Override
+    public void run(DataManager dataManager, ModuleManager moduleManager, Ui ui, RecipeManager recipeManager,
+                             Storage storage) throws DataManagerException, CommandException {
+        Data currentData = dataManager.getCurrentData();
+        if (currentData == null) {
+            throw new CommandException(ErrorMessages.INPUT_EMPTY);
+        }
+        String currentDataInput = currentData.toString();
+        ui.printInput(currentDataInput);
```
Comment (rizemon) [inline src/main/java/decodex/commands/ShowCommand.java:30]: Perhaps `ui.printOutput()` would be more fitting?

File: src/main/java/decodex/parser/Parser.java [hunk 2]
```diff
@@ -235,6 +239,16 @@ private SelectCommand prepareSelectCommand(String userInput) throws CommandExcep
         return new SelectCommand(selectCategory, itemName, parameters);
     }
 
+    /**
+     * Shows the original input data.
+     *
+     * @return The ShowCommand object.
+     * @throws CommandException If the original data input is empty.
+     */
```
Comment (rizemon) [inline src/main/java/decodex/parser/Parser.java:247]: Perhaps change this to `current data`?

File: src/main/java/decodex/ui/messages/ErrorMessages.java [hunk 3]
```diff
@@ -34,4 +34,6 @@
     public static final String RECIPE_FILE_DELETE_FAILED_MESSAGE = "Failed to delete the corresponding recipe file";
     public static final String FAILED_TO_LOAD_RECIPE_FILE_MESSAGE = " failed to be loaded into Decodex";
     public static final String INVALID_RECIPE_COMMAND = "Invalid recipe command";
+    public static final String ADDITIONAL_PARAMETER_AFTER_RECIPE_NAME_MESSAGE =
+            "Additional argument found after recipe name";
```
Comment (rizemon) [inline src/main/java/decodex/ui/messages/ErrorMessages.java:38]: A general exception like `too many arguments` would be better? I also realized we have a few `too many ...` exception messages

Comment (Kair0s3) [reply src/main/java/decodex/ui/messages/ErrorMessages.java:38]: That is literally `too many of too many`, but yes, will make changes to a more general exception.

**Review by rizemon — APPROVED** [review]
LGTM

**Review by arraysius — APPROVED** [review]
LGTM

**Pull request merged**